### DISCUSSION TITLE
What is segment size in data transfer?

### DISCUSSION THREAD

**Discussion by dastardlychimp** [dbeaver/dbeaver#20365] [Q&A] [answered]
When you export data in dbeaver you have the option to make the extract type segmented and choose a segment size. Could someone differentiate and outline what happens when segment size, fetch size, and transaction commit size is set?

![image](https://github.com/dbeaver/dbeaver/assets/12251577/0591beb1-199c-49c9-8f1a-569f2915d82c)

**Comment by E1izabeth** [comment, marked as answer]
You might find a brief explanation here https://github.com/dbeaver/dbeaver/wiki/Export-Command#producer-settings
